File: java/UI.java
/**
 * Blackjack
 * Copyright 2015 Sam Saint-Pettersen
 * Released under the MIT/X11 License.
 * 
 * Java Swing/AWT implementation.
*/

import java.awt.Graphics;
import java.awt.Color;
import java.awt.Font;

public class UI implements IUI
{   
    /**
     *  Clear a UI item.
    */
    public void clear() {}
    
    /**
     * Draw a UI item.
     * @param g Graphics object.
    */
    public void draw(Graphics g)
    {
        g.setColor(Color.WHITE);
        g.setFont(new Font("Verdana", Font.PLAIN, 12));
    }
}

File: scala/src/io/stpettersen/blackjack/UI.scala
/**
 * Blackjack
 * Copyright 2015 Sam Saint-Pettersen
 * Released under the MIT/X11 License.
 *
 * Scala Swing/AWT implementation.
 */

package io.stpettersen.blackjack
import java.awt.{Graphics,Color,Font}

class UI extends IUI {

  /**
   * Clear a UI item.
   */
  def clear: Unit = {}

  /**
   * Draw a UI item.
   * @param g Graphics object.
   */
  def draw(g: Graphics): Unit = {
    g.setColor(Color.WHITE)
    g.setFont(new Font("Verdana", Font.PLAIN, 12))
  }
}
